.pipelines/singletenancy/aks-swift/e2e-job-template.yaml

```diff
@@ -40,6 +40,9 @@ stages:
       TAG: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.Tag'] ]
       CURRENT_VERSION: $[ stagedependencies.containerize.check_tag.outputs['CurrentTagManifests.currentTagManifests'] ]
       commitID: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.commitID'] ]
+      GOPATH: "$(Agent.TempDirectory)/go" # Go workspace path
```

Comment:
I agree that commit id was shared with other stages that is why it is job level variable. Are all the other variables also need in other stages ?

Reply:
This change makes it where the variables are visible to all jobs in their respective stage.

Reply:
Understood, so the variables are visible or accessible to other jobs in the same stage. We are not using those in those jobs.